docker-compose.yml

```diff
@@ -71,26 +71,14 @@ services:
     user: 0:0
 
   zammad-elasticsearch:
-    image: bitnami/elasticsearch:${ELASTICSEARCH_VERSION:-8.18.0}
+    image: elasticsearch:${ELASTICSEARCH_VERSION:-8.18.0}
     restart: ${RESTART:-always}
     volumes:
-      - elasticsearch-data:/bitnami/elasticsearch/data
+      - elasticsearch-data:/usr/share/elasticsearch/data
     environment:
-      # See https://github.com/bitnami/containers/tree/main/bitnami/elasticsearch#environment-variables
-      #
-      # Enable authorization without HTTPS. For external access with
-      #   SSL termination, use solutions like nginx-proxy-manager.
-      ELASTICSEARCH_ENABLE_SECURITY: 'true'
-      ELASTICSEARCH_SKIP_TRANSPORT_TLS: 'true'
-      ELASTICSEARCH_ENABLE_REST_TLS: 'false'
-      # ELASTICSEARCH_USER is hardcoded to 'elastic' in the container.
-      ELASTICSEARCH_PASSWORD: ${ELASTICSEARCH_PASS:-zammad}
-      # Resource settings
-      ELASTICSEARCH_HEAP_SIZE:
-      ELASTICSEARCH_MAX_ALLOWED_MEMORY_PERCENTAGE:
-      ELASTICSEARCH_MAX_ALLOWED_MEMORY:
-      ELASTICSEARCH_MAX_TIMEOUT:
-      ELASTICSEARCH_LOCK_ALL_MEMORY:
+      discovery.type: single-node
+      xpack.security.enabled: 'false'
```

Comment:
With xpack-Security disabled does it make sense to keep ELASTICSEARCH_USER and ELASTICSEARCH_PASS? Or can this be overwritten somehow?

Reply:
They are used when you have an external ES service to connect to. Then you can disable the internal service with a scenario.